pkg/gds/admin.go

```diff
@@ -1491,3 +1486,8 @@ func (s *Admin) Available() gin.HandlerFunc {
 func (s *Admin) GetTokenManager() *tokens.TokenManager {
 	return s.tokens
 }
+
+// Routes returns the Admin API router for testing purposes.
+func (s *Admin) Routes() http.Handler {
```

Comment:
If you'd like to name it `GetRoutes` to match the other accessors, that's fine with me; I think this is a holdover from Whisper and the tests I wrote for it.